libraries/docker_service_manager_systemd.rb

```diff
@@ -19,6 +19,23 @@ class DockerServiceManagerSystemd < DockerServiceBase
     action :start do
       create_docker_wait_ready
 
+      # stock systemd unit file
```

Comment:
this shouldnt be an addition as this block was moved above the `.socket` template. you need to delete the duplicate copy on line 54.